code/confetti.js

```diff
@@ -0,0 +1,213 @@
+var confetti = {
```

Comment:
Fun function!

Reply:
According to Technigo's code guide, usage of `var` is not allowed. please use `let` or `const` 😸